react/ReadMore/ReadMore.js

```diff
@@ -86,11 +87,19 @@ class ReadMore extends PureComponent<Props, State> {
   };
 
   render() {
-    const { children, maxLines, maxRows, moreLabel, lessLabel } = this.props;
+    const {
+      children,
+      maxLines,
+      maxRows,
+      moreLabel,
+      lessLabel,
+      backgroundColor
```

Comment:
Agree about api, Name would be better, however why don't we just supply colour so it's more scalable?